### DISCUSSION TITLE
Why This is Happened for Stable Diffusion

### DISCUSSION THREAD

**Discussion by YuunaYukari** [TheLastBen/fast-stable-diffusion#870] [Q&A]
![image](https://user-images.githubusercontent.com/117509860/205497376-d0333115-b869-4e3a-9aa2-c550fa2c9d53.png)

I tried so many time but it is still have same error

**Comment by TheLastBen** [comment]
you are using an outdated colab, use this link : https://colab.research.google.com/github/TheLastBen/fast-stable-diffusion/blob/main/fast_stable_diffusion_AUTOMATIC1111.ipynb
